### DISCUSSION TITLE
January 25th, 2023

### DISCUSSION THREAD

**Discussion by LukasBuecherl** [SynBioDex/SBOL-Editors#42] [Editors meetings]
# Participants
* Chris Myers
* Prashant Vaidyanathan
* Gonzalo Vidal
* Tom Mitchell
* James Scott-Brown
* Lukas Buecherl

# Agenda

## Website (James S-B) 
* SBOL Specification 3.1.0 needs to be posted
* Combine and Harmony added

## SEPs / Github issues (Prashant)
* Focus on election

## Communications (Gonzalo)
* Monthly communication difficult
* Timely communication seems to work better
* If no updates there still need to be an update once a month

## Report on other SBOL meetings (Tom)
* SBOL 3.0 working group meeting happened 
* Roadmap for SBOL 3.0
* Four open google summer of code projects

## Any Other Business:
* Three options for voting if only one nominee (yes, no, abstain) 
* Voting will start today and will last for five business days 
* @LukasBuecherl compile updated specification and send to @Gonza10V 
* Issue with *assembly plan* seen in BP 11 and SEP 55 -> discuss to create ne activity

**Comment by Gonza10V** [comment]
I discussed the creation of the new term with @jakebeal  and he proposed another option. Instead of creating an SEP to add the term from BP011 we can create an SEP to allow BPs to add new terms to the spec. Let me know your opinion to create this new SEP. @SynBioDex/editors

**Comment by cjmyers** [comment]
Sounds like an interesting alternative.  I think we may want to create a different process for adding terms to the SBOL ontology that is streamlined, and this SEP could create that process.
